src/commands/create/contract.ts

```diff
@@ -340,7 +345,7 @@
 
             break
 
-          case 'HolographDropsEditionsV1': {
+          case 'HolographDropERC721': {
```

Comment:
should we set this as `BytecodeType.HolographDropERC72`? and other cases as well.